### DISCUSSION TITLE
Update Admin console UI

### DISCUSSION THREAD

**Discussion by pmo-van-018** [keycloak/keycloak#17270] [Archive]
Hi guys,
I need to custom the Admin Console UI, detail is I want to add new Page into the current menu, but I can not find any source base for Admin theme to custom (Just see the Account, Login, Email, Welcome). Please help me on this, thanks

**Comment by ssilvert** [comment]
The source for admin console currently resides at https://github.com/keycloak/keycloak-ui/tree/main/apps/admin-ui

However, we will soon be moving it back into the keycloak/keycloak repo.
